### PR TITLE
Weapon addon example

### PR DESCRIPTION
**Pull request by xxshady**

### CODE REVIEW

**Review by Unnvaldr — CHANGES_REQUESTED** [review]
Great thanks, would be nice to add newlines at the end of the files that doesn't have it and also replace tabs with spaces in XML files.

File: weapon-addon/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# weapon-infinityblade
```
Comment (Unnvaldr) [inline weapon-addon/README.md]: And here change to `weapon-addon`.

**Comment by xxshady** [comment]
@Unnvaldr Done.

**Review by Unnvaldr — APPROVED** [review]
Nice, thanks 😄

**Pull request merged**